src/main/java/com/google/starfish/servlets/SearchServlet.java

```diff
@@ -114,9 +54,9 @@ private String trimAndLowerCaseString(String string) {
     return string.toLowerCase().trim();
   }
 
-  /** Converts an array list to JSON */
-  private String convertListToJson(List<Note> notes) {
+  /** Converts a 2D array to JSON */
+  private String convert2DArrayToJSON(Object[][] arr) {
```

Comment:
can you share this function since you use it above, rather than re-defining it?

Reply:
Yep, I'll make a util class!